tests/integration/test_alembic.py

```diff
@@ -137,6 +145,12 @@ def test_upgrade(alembic_app):
     assert 'legacy_records_mirror' in _get_table_names()
     assert 'legacy_records_mirror_recid_seq' in _get_sequences()
 
+    # 0bc0a6ee1bc0
```

Comment:
I think you should add the upgrade step that was missing, that way, if the one that was missing fails, it will not misguide anyone into thinking that was your upgrade.